src/App.tsx

```diff
@@ -1,16 +1,34 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [visibleMovie, setVisibleMovie] = useState(moviesFromServer);
```

Comment:
better to use a copy of array to not change initial
```suggestion
  const [visibleMovie, setVisibleMovie] = useState([ ...moviesFromServer ]);
```